tests/benchmarks/test_quantum_volume.py

```diff
@@ -0,0 +1,26 @@
+from metriq_gym.benchmarks.quantum_volume import calc_stats, QuantumVolumeData
+
+
+def test_calc_stats():
+    job_data = QuantumVolumeData(
+        provider_job_id="test_job_id",
+        qubits=2,
```

Comment:
Depiste `qubits=2` being set, looking at each of the `TriaStats` objects, the number of qubits for each element is only 1 qubit. Do we want to make sure that we pass this to `TrialStat`? Otherwise, it's just going to be ignored.